waku/v2/node/peer_manager/peer_manager.nim

```diff
@@ -649,6 +698,7 @@ proc start*(pm: PeerManager) =
   asyncSpawn pm.updateMetrics()
   asyncSpawn pm.relayConnectivityLoop()
   asyncSpawn pm.prunePeerStoreLoop()
+  asyncSpawn pm.pruneConnsByIpLoop()
   asyncSpawn pm.logSummary()
```

Comment:
We have quite a large number of spawned tasks here. Wouldn't a single maintenance heartbeat be a better approach?

Reply:
yup, merged two of them in this pr
49db93e98f68c2d8c8f6258e5e7f10f66e8be764